text/000-cli-redesign.md

```diff
@@ -1 +1,115 @@
+We'd also like to make the changes outlined in the section titled below, called "List of top action items"
+A related RFC is the [Error surveying & improvement RFC](https://github.com/gatsbyjs/rfcs/pull/37)
+
+## List of first hypotheses to test
+- [ ]  If we clear the Gatsby CLI screen to only display success messages or error messages, people will save time in finding the right thing
+- [ ]  If people use a Gatsby CLI quiet mode, people will find and fix errors faster (quiet mode would make errors more discoverable)
+- [ ] If the Gatsby CLI suggests the correct spellings for misspelled Gatsby commands, people will save time and be happier. For example, when people need help, we could make it more easily discoverable so they don’t have to type multiple commands (e.g. `gatsby --help` then `gatsby build  --help`). Possible solution: provide help information for a variety of inputs `gatsby h`, `gatsby`, `gatsby help`, etc. and update our did-you-mean file to dogfood the correct commands
+- [ ] If we prevent same error message over and over again, people will be less confused and fix the error faster
+- [ ] Including createPages page count in output can help users optimize for speed and catch errors faster
+- [ ] When people type the wrong thing in the browser (e.g. “localhost:8000/_ _ graphiql), we could either redirect to "correct" one or just support multiple different cases. This will help them spend less time looking for the right link.
+- [ ] Make sure that links to development site / graphiql are easily discoverable, which will also help make sure users can find the links without much scrolling. The problem is that errors / recompiling notices are getting appended endlessly so users need to scroll very far sometimes to get to those links. Possible solutions: If we would clear older errors / notices so the links are few lines away would fix it as well. Another option is for `gatsby develop` to be interactive (like `jest --watch`) where you can use commands while it's running and we could have command to open browser with development site / graphiql as part of the available commands. With ink we can make sure we don't add too much info at the bottom.
+- [ ] Offering suggestions for those who want to customize CLI commands, such as `gd` for `gatsby develop` could help people stick with gatsby because they can customize it, make it theirs, and get stuff done faster.
+- [ ] If we shorten the effort needed to input a starter URL into the `gatsby new` command, people will be happy because it’s easier. One idea is to link to starter library or give options of starter in the CLI.
+Shortening the input needed is nice: you can take away the https://www before the starter name and it still worked
+- [ ] Get rid of babel stack traces for errors and no one will miss them and will have an easier time finding errors that matter to them
```

Comment:
What does that exactly mean? (what are babel stack traces?)

Reply:
I'm not sure; these are from notes I took during usability tests! It's possible we wrote something incorrectly.

Reply:
it's from internal errors that throw stack traces but where the stack trace is for e.g. babel parsing and not helpful to the user. I'll clarify it.